api/openapi_server/models/database.py

```diff
@@ -1,14 +1,26 @@
-from sqlalchemy import create_engine
+from sqlalchemy import create_engine, Table, text
 from sqlalchemy.engine import Engine
-from sqlalchemy.orm import Session, declarative_base
+from sqlalchemy.orm import Session, declarative_base, relationship
 from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.exc import SQLAlchemyError
 
 Base = declarative_base()
 
 class User(Base):
     __tablename__ = "user"
     id = Column(Integer, primary_key=True, index=True)
     email = Column(String, nullable=False, unique=True)
+    firstName = Column(String(255), nullable=False)
+    middleName = Column(String(255), nullable=True)
+    lastName = Column(String(255), nullable=False)
+    role_id = Column(Integer, ForeignKey('role.id'), nullable=False)
+    role = relationship("Role", back_populates="users")
```

Comment:
Does this return the role as a string on the user object?

Reply:
It returns a `models.database.Role` object. The `relationship` function uses the `role_id` to perform an automatic lookup of the `Role` object whenever the `user` object is accessed. It'll also perform some validation when updating the `role_id` to make sure it is a valid `Role`. 

The `User` table doesn't actually store a `role` - just the `role_id`: 
![image](https://github.com/hackforla/HomeUniteUs/assets/22138019/7db6d020-bfa9-4b36-9e8a-7526f15e394f)

But if you dump the `user` model to json you see the serialized `role` object:
`{'role': {'name': 'Admin'}, 'email': 'testcoordinator@test.com', 'firstName': 'testuser_firstname', 'middleName': '', 'lastName': 'testuser_lastname'}`

Reply:
Okay cool. If that's the object being returned when the user logs in then that's perfect. Just wanna make sure we have the role on the front end so we can do things like render routes based on role, show different components, etc.